workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/MapQualitySettings.cs

```diff
@@ -3,35 +3,45 @@
 using Fps;
 using UnityEngine;
 
-public class TileSettings : MonoBehaviour
+public class MapQualitySettings : MonoBehaviour
 {
-    [Tooltip("Only functional in editor")] public bool ForceIsClient;
+    public bool Preview;
 
-    private static TileSettings Instance;
     public const float DefaultCheckoutDistance = 300f;
     public const float DefaultCheckoutDistanceSquared = 300f * 300f;
     public static float CheckoutDistance => Instance.CheckoutDistanceInternal;
```

Comment:
would it make sense to switch to a ScriptableObject for this? 
Using an `Instance` in a static field could cause problems as soon as you have the MonoBehaviour multiple times in the scene as you can't predict which Instance is gonna be used in the end

Reply:
Okay, just merged in a branch that uses the publisher pattern to specify the MapQualitySettings. It's quite a hefty change, and I fixed a number of other issues as well (see commit description for details). Kinda a two steps forward one step back deal, but I hope it's a big improvement on usage quality.